### PR TITLE
[FIX] payment_paypal: correct dead link

### PR DESCRIPTION
**Pull request by Maralai**

### CODE REVIEW

File: addons/payment_paypal/views/payment_provider_views.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
                     <field name="paypal_pdt_token" groups="base.group_no_one" password="True"/>
                     <field name="paypal_use_ipn"
                            attrs="{'required':[('code', '=', 'paypal'), ('state', '!=', 'disabled')]}"/>
-                    <a href="https://www.odoo.com/documentation/16.0/applications/general/payment_providers/paypal.html"
+                    <a href="https://www.odoo.com/documentation/master/applications/finance/payment_providers/paypal.html"
```
Comment (Demesmaeker) [inline addons/payment_paypal/views/payment_provider_views.xml:20]:
```suggestion
                    <a href="https://www.odoo.com/documentation/16.0/applications/finance/payment_providers/paypal.html"
```

**Review by Demesmaeker — COMMENTED** [review]
Hello, thanks for your catch and input! 
As this PR target the V16.0, could you keep this version instead?
Thanks again, and have a nice day,

**Review by Demesmaeker — APPROVED** [review]

**Comment by Demesmaeker** [comment]
It's okay, you can follow this guide to do so: https://github.com/odoo/odoo/blob/16.0/doc/cla/sign-cla.md
Have a nice day,

**Comment by Maralai** [comment]
Done!

**Comment by d-fence** [comment]
robodoo r-

**Pull request closed without merging**